src/PublisherMethods.js

```diff
@@ -103,5 +105,37 @@ module.exports = {
         _.nextTick(function() {
             me.trigger.apply(me, args);
         });
-    }
+    },
+
+    /**
+     * Returns a Promise for the triggered action
+     */
+    triggerPromise: function(){
+        var me = this;
+        var args = arguments;
+
+        var canHandlePromise =
+            this.children.indexOf('completed') >= 0 &&
+            this.children.indexOf('failed') >= 0;
+
+        if (!canHandlePromise){
+            throw new Error('Publisher must have "completed" and "failed" child publishers');
+        }
+
+        var promise = new Promise(function(resolve, reject) {
```

Comment:
I agree.  Personally, I'm a fan of [Bluebird][1], but it seemed wise to default to a polyfill at least, since [ES6 Promises are garnering adoption][2].

For the sake of symmetry, perhaps `Reflux.setPromise(...)` (similar to `Reflux.setEventEmitter`) would be best?

The README would look like this...

> ### Switching the Promise library
>
> Don't like to use the Promise library provided? You can switch to another one, such as [Bluebird](https://github.com/petkaantonov/bluebird/) like this:
>
> ```javascript
> // Do this before triggering actions
>
> Reflux.setPromise(require('bluebird'));
> ```
>

Let me know if this solution suits you & I'll push the code, otherwise I'll do your `createPromise` recommendation (which seems like more work, but it'd be worth having _both_ solutions for libraries that follow the spec and users who want to shim in a library that _does not_.).

[1]: https://github.com/petkaantonov/bluebird/
[2]: http://kangax.github.io/compat-table/es6/#Promise